tests/wrappers/test_atari_preprocessing.py

```diff
@@ -1,90 +1,118 @@
 import numpy as np
 import pytest
 
-import gym
+from gym.spaces import Box, Discrete
 from gym.wrappers import AtariPreprocessing
+from tests.testing_env import GenericTestEnv, old_step_fn
 
-pytest.importorskip("gym.envs.atari")
 
+class AleTesting:
+    grayscale_obs_space = Box(low=0, high=255, shape=(210, 160), dtype=np.uint8, seed=1)
+    rgb_obs_space = Box(low=0, high=255, shape=(210, 160, 3), dtype=np.uint8, seed=1)
 
-@pytest.fixture(scope="module")
-def env_fn():
-    return lambda: gym.make("PongNoFrameskip-v4", disable_env_checker=True)
+    def lives(self):
+        return 1
 
+    def getScreenGrayscale(self, buffer):
+        buffer[...] = self.grayscale_obs_space.sample()
 
-def test_atari_preprocessing_grayscale(env_fn):
-    import cv2
+    def getScreenRGB(self, buffer):
+        buffer[...] = self.rgb_obs_space.sample()
 
-    env1 = env_fn()
-    env2 = AtariPreprocessing(
-        env_fn(), screen_size=84, grayscale_obs=True, frame_skip=1, noop_max=0
-    )
-    env3 = AtariPreprocessing(
-        env_fn(), screen_size=84, grayscale_obs=False, frame_skip=1, noop_max=0
-    )
-    env4 = AtariPreprocessing(
-        env_fn(),
+
+class AtariTestingEnv(GenericTestEnv):
+    """Testing environment to replicate the ale-py environment."""
+
+    def __init__(self):
+        super().__init__(
+            observation_space=Box(
+                low=0, high=255, shape=(210, 160, 3), dtype=np.uint8, seed=1
+            ),
+            action_space=Discrete(4, seed=1),
+            step_fn=old_step_fn,
+        )
+        self.ale = AleTesting()
+
+    def get_action_meanings(self):
+        """Testing function."""
```

Comment:
Docstring a little more descriptive

Reply:
Done in c6afb8e05986820df636769a15d6330ce475c9ff